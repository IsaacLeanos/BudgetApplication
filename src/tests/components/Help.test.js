import React from 'react'
import{shallow}from'enzyme'
import HelpExpensePage from '../../components/HelpExpense'


test('renderer Help component',()=>{
    let wrapper=shallow(<HelpExpensePage/>)
    expect(wrapper).toMatchSnapshot()
})