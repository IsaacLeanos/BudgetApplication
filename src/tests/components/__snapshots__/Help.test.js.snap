// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renderer Help component 1`] = `
<div>
  HELP
</div>
`;
